### PR TITLE
Added Privacy Policy blurb to radix forms

### PR DESCRIPTION
**Pull request by CaitlynKnudsvig**
Added "I accept that the data provided on this form will be processed, stored and used in accordance with the terms set in our privacy policy" to the email subscription form, gated download forms, and registration form globally

screen shot 2018-08-21 at 9 54 08 am
screen shot 2018-08-21 at 8 21 53 am
screen shot 2018-08-21 at 8 47 49 am

### CODE REVIEW

File: src/AppBundle/Resources/library/js/components/register.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 React.createClass({ displayName: 'ComponentRegister',
   getDefaultProps: function() {
     return {
-      title: 'Register',
-      description: null,
+      title: 'Title here - Register',
+      description: 'Description Goes Here',
```
Comment (solocommand) [inline src/AppBundle/Resources/library/js/components/register.js]: These should probably get reverted :)

File: src/AppBundle/Resources/library/js/components/register.js [hunk 2]
```diff
@@ -139,13 +144,15 @@ React.createClass({ displayName: 'ComponentRegister',
...
-      });
+      form = React.createElement('div', null,
+        React.createElement(Radix.Components.get('Form'), {
+          name: 'register',
+          fields: this.state.fields,
+          values: this.state.values,
+          onChange: this.updateFieldValue,
+          onSubmit: this.handleSubmit
+        })
+      );
```
Comment (solocommand) [inline src/AppBundle/Resources/library/js/components/register.js]: Why are you wrapping the form in an extra div?

**Comment by solocommand** [comment]
Also, in the future this repository should be forked and have PRs submitted from your fork, rather than creating a branch.

**Comment by CaitlynKnudsvig** [comment]
@solocommand https://southcomm.atlassian.net/browse/CS-55
I'll make those changes quick

**Comment by solocommand** [comment]
@CaitlynKnudsvig did you see my questions about the jira ticket and text changes?

**Comment by CaitlynKnudsvig** [comment]
@solocommand yea here's the actual ticket, which is where the verbiage and lack of a checkbox stems from: https://southcomm.atlassian.net/browse/CS-55

**Comment by solocommand** [comment]
@CaitlynKnudsvig that being the case, this should either use the updated text that is more semantically clear, or add this exact text only for AC applications, similar to the support contact information that is already being customized on every group.

Really, the latter should be done regardless, as every site using Radix may not have a privacy policy.

**Comment by CaitlynKnudsvig** [comment]
@solocommand *shrug* I can update it quick, gimme a min

**Pull request merged**